Library/Homebrew/tap_auditor.rb

```diff
@@ -10,12 +10,14 @@ class TapAuditor
     sig { params(tap: Tap, strict: T.nilable(T::Boolean)).void }
     def initialize(tap, strict:)
       Homebrew.with_no_api_env do
+        tap.clear_cache
```

Comment:
Not sure if this is right way, but looks like something has already populated the local cache perhaps with API environment.

Clearing it finally gets CI to show audit failure.